jerry-core/ecma/operations/ecma-conversion.cpp

```diff
@@ -489,9 +489,6 @@ ecma_op_from_property_descriptor (const ecma_property_descriptor_t* src_prop_des
   else
   {
     // 4.
-    JERRY_ASSERT (src_prop_desc_p->is_get_defined
-                  || src_prop_desc_p->is_set_defined);
```

Comment:
@kkristof, the assertion is specification-defined.

http://www.ecma-international.org/ecma-262/5.1/#sec-8.10.4, item 4:
`IsAccessorDescriptor(Desc) must be true`

It seems to me that actual issue is in `ecma_get_property_descriptor_from_property` function: instead of putting `undefined` value to [[Get]] and [[Set]] fields, the function sets the fields as undefined.